src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -145,6 +145,15 @@ public static LocalizableErrorArgument Localize(this MessageID id)
             return new LocalizableErrorArgument(id);
         }
 
+        internal static string RequiredFeature(this MessageID feature)
```

Comment:
Nit: would named this to indicate an action is occuring.  For example ConvertToRequiredFeatureId.